frontend/drink_list_picker.js

```diff
@@ -1,11 +1,18 @@
 "use strict";
 
-
-
-async function login_user() {
+async function all_drink_choices() {
+    let drink_list_page = document.getElementById("drink_list_page");
+    let created_list = document.getElementById("drink_list_show");
+    try {
+        drink_list_page.removeChild(created_list);
+    }
+    catch {
```

Comment:
Why is there a try/catch? Do you expect an error? If so, why is there nothing in the catch statement?
And, if you do expect an error, you should try to be consistent and use a try/catch everywhere you remove a child :)